README.md

```diff
@@ -1,6 +1,8 @@
-# Portfolio Mobile First Setup
+# Portfolio: finish mobile version
 
 > This project has been built as part of the reviewers exercise or training.
+> Build the middle section of the mobile website. 
+> The "About myself" section.
 
 ## Built With
 
```

Comment:
- [ ] kindly add the `get starting` section to your readme file, to describe how to use & setup your project